charts/template-api/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm base-chart for Kubernetes to deploy the api
+name: template-api
+version: 0.1.0
+keywords:
```

Comment:
The keywords is used in repository UI / CLI search command. You can add 
- symfony 
- template 
...